app/views/register/trustees/trusteeKind.scala.html

```diff
@@ -38,7 +38,7 @@
 
         @components.input_radio(
             field = form("value"),
-            legend = messages("messages__trusteeKind__legend"),
+            legend = "",
```

Comment:
Why the empty legend?

Reply:
after a discussion earlier it was agreed that the legend will be incorporated into the h1 at a later date so for now the legend should be empty

Reply:
This still creates an empty span element which is invalid HTML. I agree with your comment, legend should be incorporated into the H1 where there is no other question/lede text on the page, but until that is implemented legend should still be populated.

Reply:
legend updated as visually hidden text

Reply:
Checked - text reinstated.